platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/inputs/undeclared/UndeclaredBuildInputsKotlinBuildSrcIntegrationTest.groovy

```diff
@@ -24,6 +24,11 @@ class UndeclaredBuildInputsKotlinBuildSrcIntegrationTest extends AbstractUndecla
         return "Plugin class 'SneakyPlugin'"
     }
 
+    @Override
+    protected String getSysProp() {
+        return "PROPY" // KGP's BuildFusService uses the default "CI" property and messes up these tests
```

Comment:
💅 We could use a more unique name for the property for all tests instead of space-casing just the Kotlin ones because they happen to read the one we currently use in the test.

Reply:
Yes, I've considered that and I've decided to not modify everything for a Kotlin specific issue, but yes, at a second thougth I might have picked the worse solution. Will adjust.